test/lib/tasks/seeds_test.rb

```diff
@@ -9,6 +9,7 @@ class SeedsTest < ActiveSupport::TestCase
     DatabaseCleaner.clean_with :truncation
     Setting.stubs(:[]).with(:administrator).returns("root@localhost")
     Setting.stubs(:[]).with(:send_welcome_email).returns(false)
+    Setting.stubs(:[]).with('entries_per_page').returns(20)
```

Comment:
I think everything comes from the fact that https://github.com/theforeman/foreman/blob/develop/lib/foreman.rb#L16 is not really detecting whether you're on a rake task or not. There's another PR I opened about this - https://github.com/theforeman/foreman/pull/2797 I should reopen with a proper fix.